### PR TITLE
Music

### PR DESCRIPTION
**Pull request by dsmarcot2018**
Added music functionality and updated the README to be more accurate. Please give this a test and make sure that it works.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,44 +1,63 @@
...
+* Hunter Spack
+* Drew Marcotte
+* Joe Marchesini
+
+## Bot Set Up
+1. Open the bot folder as a project in PyCharm and then set up a virtual environment for your bot.
+2. Run the command "pip install -U discord.py" in your command line.
+3. Run the command "pip install -U python-dotenv" in your command line.
+4. Run the command "pip install --upgrade youtube_dl" in your command line.
+5. Run the command "pip install PyNaCl" in your command line.
```
Comment (Joe-Marchesini) [inline README.md:14]: Update to reflect my new format of readme listing the pip installs in its own section

**Comment by RayZhengGit** [comment]
Fixed the merge conflicts

**Review by Joe-Marchesini — DISMISSED** [review]
Address readme issue

**Review by RayZhengGit — APPROVED** [review]
Good to go here we went over this together last week!

**Review by Joe-Marchesini — APPROVED** [review]

**Review by xSpacklesx — APPROVED** [review]

**Pull request merged**